qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -272,29 +255,72 @@ def defaults(self, refresh: bool = False) -> Optional[PulseDefaults]:
 
         return self._defaults
 
-    def jobs_limit(self) -> BackendJobsLimit:
-        """Return the jobs limit for the backend.
+    def job_limit(self) -> BackendJobLimit:
+        """Return job limit for the backend.
+
+        The job limit information may include the current number of
+        jobs running on the backend at this time and the maximum number
+        of concurrent jobs a user could submit to the backend at a time.
 
         Note:
-            If `BackendJobsLimit.maximum_jobs = -1`, then there
-            are no limits to the maximum number of jobs that could
-            be submitted on the backend.
+            The job limit information for the backend is given for
+            this account (i.e. a specific backend with a specific
+            provider).
+
+            If the method call was successful, you can inspect the job
+            limit for the backend by accessing the ``maximum_jobs``
+            and ``running_jobs`` attributes of the ``BackendJobLimit``
+            class returned.
+
+            For example:
+                backend_job_limit = IBMQBackend.job_limit()
+                maximum_jobs = backend_job_limit.maximum_jobs
+                running_jobs = backend_job_limit.running_jobs
+
+            * If ``maximum_jobs`` or ``running_jobs`` are ``None``, the
+                job limit information is currently not available.
+            * If ``maximum_jobs`` is equal to ``-1``, then there are
+                no limits to the maximum number of concurrent jobs a user
+                could submit to the backend at a time.
 
         Returns:
-            the current number of running jobs and the maximum number
-            of jobs for the backend.
+            the current number of jobs running on the backend and the
+            maximum number of concurrent jobs a user could submit to the
+            backend at a time.
 
         Raises:
             LookupError: If jobs limit for the backend can't be found.
         """
-        api_jobs_limit = self._api.backend_jobs_limit(self.name())
+        api_job_limit = self._api.backend_job_limit(self.name())
 
         try:
-            return BackendJobsLimit.from_dict(api_jobs_limit)
+            return BackendJobLimit.from_dict(api_job_limit)
         except ValidationError as ex:
             raise LookupError(
                 "Couldn't get backend jobs limit: {0}".format(ex))
 
+    def remaining_jobs(self) -> int:
+        """Return remaining jobs that could be submitted to the backend.
+
+        Return the remaining number of jobs that could be submitted to the
+        backend, for this account, before the job limit is reached. See
+        ``IBMQBackend.job_limit()`` for the job limit of a given backend.
```

Comment:
I'd avoid using the word `account`, since an account can have multiple providers. Using `provider` is more accurate. Also, I'd add the word `concurrent` to the description since the current wording makes it sound like the limit is for all jobs (running or finished). Maybe something like

`Return the number of jobs that can be submitted to this backend with this provider before the limit on concurrent jobs is reached.`

This applies to the rest of the docstring.